### PR TITLE
Add a dawn backend flag to the script

### LINKED ISSUE #567: Add a file or script, listing all the tests that pass in Dawn backend

### PR DESCRIPTION
**Pull request by sarahM0**
fixes #567

### CODE REVIEW

File: tests/run_tests.py [hunk 1]
```diff
@@ -88,6 +92,8 @@ def RunTest(self, tc):
     cmd = [self.options.test_prog_path, '-q']
     if tc.IsParseOnly():
       cmd += ['-p']
+    if tc.IsUseDawn():
+      cmd += ['-e'] +  ['dawn']
```
Comment (dneto0) [inline tests/run_tests.py]: This is fine.  But it could have been   

     cmd += ['-e', 'dawn']

File: tests/run_tests.py [hunk 2]
```diff
@@ -52,12 +52,89 @@
     # DXC not currently building on bot
     "draw_triangle_list_hlsl.amber"
    ]
-}
+ }
+
+SUPPRESSIONS_DAWN = [
+  "compute_accumulated_ubo_definition.amber",
```
Comment (dj2) [inline tests/run_tests.py]: Can you add a comment as to why things are suppressed? Grouping them is fine and a single #not implemented is also fine. Just, if there are long lived suppressions knowing why is handy.

File: tests/run_tests.py [hunk 3]
```diff
@@ -126,6 +213,10 @@ def SummarizeResults(self):
       for failure in self.failures:
         print failure
 
+      print '\nSummary of Successes:'
```
Comment (dj2) [inline tests/run_tests.py]: I'm not sure if we want to add this, printing the successes is going to hide the failures.

Comment (sarahM0) [reply tests/run_tests.py]: I'll remove it. I added it before I add the suppression list to see what is passing and now there is no need for it.

**Pull request merged**